tensorflow_datasets/image/lfw.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Labeled faces in wild."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+
+_URL = "http://vis-www.cs.umass.edu/lfw/lfw.tgz"
+
+LFW_IMAGE_SHAPE = (250, 250, 3)
+
+LFW_CITATION = """\
+@TechReport{LFWTech,
+    author = {Gary B. Huang and Manu Ramesh and Tamara Berg and Erik Learned-Miller},
+    title = {Labeled Faces in the Wild: A Database for Studying Face Recognition in Unconstrained Environments},
+    institution = {University of Massachusetts, Amherst},
+    year = 2007,
+    number = {07-49},
+    month = {October}
+}
+"""
+
+
+class LFW(tfds.core.GeneratorBasedBuilder):
+  """LFW Class"""
+  URL = "http://vis-www.cs.umass.edu/lfw/#resources"
+  VERSION = tfds.core.Version("0.1.0")
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=("Labeled Faces in the Wild:\
+  A Database for Studying Face Recognition in Unconstrained Environments"),
+        features=tfds.features.FeaturesDict({
+            "image_name": tfds.features.Text(),
```

Comment:
`label` could be better thane `image_name` to consistency and meaning.

Reply:
@us there is no label here in this dataset. it's just dir by the name of a person in which there are pictures of the person, and hence the most logical thing would be to give the name of the image and the image itself. But if you have something else in mind please do suggest.